### PR TITLE
Fixes issue #1873 (proper pull request)

### LINKED ISSUE #1873: Indeed Jobs: Implement new footer design
We now have an optional `datebadge`property that can be specified for each tile's `item` (should be specified using the `normalize` function.

We need to add a `datebadge` to the Jobs Spice to better display the posting date for each job.

Live examples can be seen in the SeatGeek Spice: https://duckduckgo.com/?q=concerts+in+toronto&ia=concerts

Code example: https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/seat_geek/events_by_city/seat_geek_events_by_city.js#L153

------
IA Page:  http://duck.co/ia/view/indeed_jobs

### PR DESCRIPTION
**Pull request by cinlloc**
Fixes #1873. Should fix both **Indeed Jobs** and **Sublime** design.

_This pull request is a fix for pull request #2229 which was not properly done (contained unwanted commits)_

### CODE REVIEW

**Comment by daxtheduck** [comment]
@cinlloc Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

File: share/spice/sublime_packages/footer.handlebars [hunk 1]
```diff
@@ -1,4 +1,10 @@
 <div class="tile__update__info one-line">
-	<i class="ddgsi ddgsi-arrow-down"></i>
-	{{unique_installs}} &middot; Updated {{last_modified}}
+        <span class="opt--left">
```
Comment (moollaza) [inline share/spice/sublime_packages/footer.handlebars]: @cinlloc It looks like our `opt--left` and `opt--right` classes aren't working right now. I'll switch these to `pull-left`, `pull-right` which do work.

**Comment by moollaza** [comment]
@cinlloc Thanks! This LGTM :+1:

File: share/spice/sublime_packages/footer.handlebars (hunk 1 above)
Comment (moollaza) [inline share/spice/sublime_packages/footer.handlebars]: Ah I understand now that the `opt--*` only take effect on small screens, so we want to use pull--* for it to affect all screens.

**Pull request merged**